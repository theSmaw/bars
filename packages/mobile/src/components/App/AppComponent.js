import React from 'react';
import styled from 'styled-components/native';

import BarsComponent from '../Bars/BarsComponent'; // eslint-disable-line import/no-named-as-default

export default function AppComponent() {
  return (
    <Container>
      <BarsComponent />
    </Container>
  );
}

const Container = styled.View`
  flex: 1;
  justify-content: center;
  align-items: center;
  background-color: #fbf7f2
`;
