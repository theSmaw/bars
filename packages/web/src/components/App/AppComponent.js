import React from 'react';

import BarsComponent from '../Bars/BarsComponent'; // eslint-disable-line import/no-named-as-default

export default function AppComponent() {
  return (
    <div className="app">
      <BarsComponent />
    </div>
  );
}
