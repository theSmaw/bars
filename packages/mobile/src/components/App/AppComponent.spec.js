import React from 'react';
import { shallow } from 'enzyme';

import AppComponent from './AppComponent';

describe('AppComponent', () =>

  it('Should render', () => {
    const appComponent = shallow(<AppComponent />);

    expect(appComponent).toMatchSnapshot();
  })
);
