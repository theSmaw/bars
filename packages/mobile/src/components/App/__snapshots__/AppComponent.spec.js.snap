// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent Should render 1`] = `
<Styled(Component)>
  <Connect(BarsContainer) />
</Styled(Component)>
`;
